### PR TITLE
Morse and mesh and zenoh

### PR DESCRIPTION
**Pull request by jr1221**
## Changes

_Explanation of changes goes here_

- Morse final inclusion
- Mesh setup and config
- Zenoh packaging and config (stage 1)
- NanoMQ fixups
- wheel-buttons fixups
- Version bumps

## Notes

_Any other notes go here_

## Test Cases

- Case A
- Edge case
- ...

## To Do

_Any remaining things that need to get done_

- [ ] item 1
- [ ] ...

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please reach out to your Project Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] All commits are tagged with the ticket number
- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes # (issue #)

### CODE REVIEW

File: odysseus_tree/overlays/rootfs_overlay_mesh/etc/dhcpcd.conf [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+denyinterfaces lo
+
+# wlan1 is the 2.4 GHz
+interface wlan1
+    static ip_address=192.168.0.1/24
+    nohook wpa_supplicant
+
+# wlan0 is the 900 MHz
+interface wlan0
+    static ip_address=10.42.0.X/24
```
Comment (caiodasilva2005) [inline odysseus_tree/overlays/rootfs_overlay_mesh/etc/dhcpcd.conf:22]: so does including X here just tell dhcp to give any available IP under the subnet here?

File: odysseus_tree/overlays/rootfs_overlay_tpu/etc/dhcpcd.conf [hunk 2]
```diff
@@ -26,6 +26,10 @@ interface wlan0
     nohook resolv.conf, mtu, ntp
     metric 300
 
+# STP 802.3
+interface eth1
+    static ip_address=10.0.0.1/24
```
Comment (caiodasilva2005) [inline odysseus_tree/overlays/rootfs_overlay_tpu/etc/dhcpcd.conf:31]: is this eth to router? how did that work without this before?

File: .github/workflows/build_image.yml [hunk 3]
```diff
@@ -53,8 +53,11 @@ jobs:
...
             ODY_IROH_ROOT_PASSWORD: ${{ secrets.ODY_IROH_ROOT_PASSWORD }}
+            ODY_MESH_ROOT_PASSWORD: ${{ secrets.ODY_MESH_ROOT_PASSWORD }}
+            ODY_MESH_AP_PASSWORD: ${{ secrets.ODY_MESH_AP_PASSWORD }}
+            ODY_MESH_SAE_PASSWORD: ${{ secrets.ODY_MESH_SAE_PASSWORD }}
             MASTER_PASSWORD: ${{ secrets.ODY_MASTER_PASSWORD }}
-        run:  for i in ODY_AP_ROOT_PASSWORD ODY_TPU_ROOT_PASSWORD ODY_WHEEL_ROOT_PASSWORD ODY_WHEEL_AP_PASSWORD ODY_IROH_ROOT_PASSWORD ODY_BASE_WIFI_PASSWORD ODY_BASE_FIELD_WIFI_PASSWORD; do echo "$i=${!i}" >> SECRETS.env; done; echo "" >> SECRETS.env; gpg --batch --symmetric --passphrase "$MASTER_PASSWORD" --no-symkey-cache --cipher-algo AES256 SECRETS.env
+        run:  for i in ODY_AP_ROOT_PASSWORD ODY_TPU_ROOT_PASSWORD ODY_WHEEL_ROOT_PASSWORD ODY_WHEEL_AP_PASSWORD ODY_IROH_ROOT_PASSWORD ODY_MESH_ROOT_PASSWORD ODY_MESH_AP_PASSWORD ODY_MESH_SAE_PASSWORD ODY_BASE_WIFI_PASSWORD ODY_BASE_FIELD_WIFI_PASSWORD; do echo "$i=${!i}" >> SECRETS.env; done; echo "" >> SECRETS.env; gpg --batch --symmetric --passphrase "$MASTER_PASSWORD" --no-symkey-cache --cipher-algo AES256 SECRETS.env
```
Comment (caiodasilva2005) [inline .github/workflows/build_image.yml:60]: can you explain how secrets work again? there a local secrets file that the container reads from and then just stores in env or something like that right?

File: odysseus_tree/overlays/rootfs_overlay_mesh/etc/dhcpcd.conf (hunk 1 above)
Comment (jr1221) [reply odysseus_tree/overlays/rootfs_overlay_mesh/etc/dhcpcd.conf:22]: there is a script that replaces it with which node ID it is.  https://github.com/Northeastern-Electric-Racing/Odysseus/blob/8ccff923b55bb54a17d26b55978f25427552337a/odysseus_tree/post-build-mesh.sh.  It uses the ODY_MESH_ID which should be passed as an env var.

File: odysseus_tree/overlays/rootfs_overlay_tpu/etc/dhcpcd.conf (hunk 2 above)
Comment (jr1221) [reply odysseus_tree/overlays/rootfs_overlay_tpu/etc/dhcpcd.conf:31]: it did not I forgot to commit it :skull:

File: .github/workflows/build_image.yml (hunk 3 above)
Comment (jr1221) [reply .github/workflows/build_image.yml:60]: https://nerdocs.atlassian.net/wiki/spaces/NER/pages/1782513679/Odysseus+and+Siren+Guide#Passwords
